runtime/src/bank/tests.rs

```diff
@@ -12116,7 +12116,8 @@ fn test_runtime_feature_enable_with_program_cache() {
         &feature_set::reject_callx_r10::id(),
         &feature::create_account(&Feature { activated_at: None }, feature_account_balance),
     );
-    bank.apply_feature_activations(ApplyFeatureActivationsCaller::NewFromParent, false);
+    goto_end_of_slot(&bank);
+    let bank = new_from_parent(Arc::new(bank));
```

Comment:
This test needs to trigger a rerooting here, so that `LoadedPrograms::prune()` gets called.

Reply:
Yes, that makes sense. I believe the test was not fully functional since the recompilation functionality was still under works.